README.md

```diff
@@ -30,33 +30,24 @@ Clone the repository.
 
 | Directory  | Chapter |
 | ------------- | ------------- |
-| Chapter 1 - Introduction | src/main/scala/com/professionalscala/ch1 |
-| Chapter 2 - Functional Programming | src/main/scala/com/professionalscala/ch2 |
-| Chapter 3 - Concurrency | src/main/scala/com/professionalscala/ch3 |
-| Chapter 3 - Scala and Java Collections | src/main/scala/com/wrox/professionalscala/ch03 |
+| Chapter 1 - Introduction | src/main/scala/com/wrox/professionalscala/ch01 |
+| Chapter 2 - Functional Programming | src/main/scala/com/wrox/professionalscala/ch02 |
+| Chapter 3 - Java Compatibility | src/main/scala/com/wrox/professionalscala/ch03 |
 | Chapter 4 - SBT | chapter4_codesamples |
 | Chapter 5 - Maven | chapter-5 |
 | Chapter 7 - Testing | chapter7_codesamples |
 | Chapter 8 - Scaladoc | chapter-8 |
 | Chapter 9 - Type System | src/main/scala/com/wrox/professionalscala/ch09 |
 | Chapter 10 - Advanced Functional Programming | src/main/scala/com/wrox/professionalscala/ch10 |
-| Chapter 11 - Java Compatibility | src/main/scala/com/professionalscala/ch11 |
+| Chapter 11 - Java Compatibility | src/main/scala/com/wrox/professionalscala/ch11 |
 
 ## Usage
 
-### Chapters 1, 2, 4, 7, 11 and some of Chapter 3
+### Chapters 1, 2, 3, 4, 7, 8, 9, 10 and 11
 
 Run the examples using `sbt` on the  command line (sbt run  will do the trick).
 
-### Chapters 9, 10 and some of Chapter 3
```

Comment:
Was meant to be included in the sense that I removed the `runMain` part and left only the `sbt run` part which seems to be OK for everyone.